app/models/workflow.rb

```diff
@@ -138,7 +138,7 @@ def extractors_runner
 
   def rerun_extractors(duration = 3.hours)
```

Comment:
Let's talk about this default duration. I realize why it's spaced out so much, but 3 hours is long enough to make this button basically useless for testing, which is the only time I've ever used it. Are careless users with giant workflows a danger in this part of the UI?

Reply:
The button doesn't warn you. That said, we could base the duration on the number of records it needs to do?

Reply:
Cool, I'll write up an issue for this.